### PR TITLE
Update Hadoop log artifacts

### PR DESCRIPTION
**Pull request by Fryyyyy**
Will catch recently observed attacks

### CODE REVIEW

**Comment by binglot** [comment]
@joachimmetz - we may need your help here ;) the unit tests throw errors about the GLOB:

```
WARNING:root:Unsupported path: /**2/hadoop/**1/yarn/system/rmstore/FSRMStateRoot/RMAppRoot/application_*/application_* with multiple globstars defined by artifact definition: HadoopAppRoot in file: data\hadoop.yaml
WARNING:root:Unsupported path: /**2/hadoop/**1/yarn/timeline/leveldb-timeline-store.ldb/* with multiple globstars defined by artifact definition: HadoopYarnLogs in file: data\hadoop.yaml
```

I wonder if there's a better way to do those GLOBs or if the unit tests need tweaking.

**Comment by joachimmetz** [comment]
Isn't `**1` the same as `*` ?

**Comment by Fryyyyy** [comment]
Seems like that fixed it :)

**Comment by joachimmetz** [comment]
makes sense the validation fails on use of multiple globstars `**` in the same path

File: data/hadoop.yaml [hunk 1]
```diff
@@ -4,5 +4,37 @@ name: HadoopAppRoot
...
 sources:
 - type: FILE
-  attributes: {paths: ['/hadoop/yarn/system/rmstore/FSRMStateRoot/RMAppRoot/application_*/application_*']}
+  attributes:
+    paths:
+    - '/hadoop/yarn/system/rmstore/FSRMStateRoot/RMAppRoot/application_*/application_*'
+    - '/**2/hadoop/yarn/system/rmstore/FSRMStateRoot/RMAppRoot/application_*/application_*'
+    - '/hadoop/*/yarn/system/rmstore/FSRMStateRoot/RMAppRoot/application_*/application_*'
+    - '/**2/hadoop/*/yarn/system/rmstore/FSRMStateRoot/RMAppRoot/application_*/application_*'
+labels: [Hadoop]
```
Comment (joachimmetz) [inline data/hadoop.yaml:13]: Labels are no longer used

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**